Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -1140,7 +1140,7 @@ private function retrieveFrontendSubRequestResult(
      * + false: always deny, e.g. when importing data is finished
      *
      * @param bool|null $allowIdentityInsert
-     * @throws DBALException
+     * @throws \Doctrine\DBAL\Exception
```

Comment:
Generally for `@throws` annotations, it is not required to use a FQCN. So we can use `@throws DBALException` in all places.

Reply:
Done.